main.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/hashicorp/packer-plugin-vsphere/builder/vsphere/iso"
 	"github.com/hashicorp/packer-plugin-vsphere/builder/vsphere/supervisor"
 	"github.com/hashicorp/packer-plugin-vsphere/post-processor/vsphere"
-	vsphere_template "github.com/hashicorp/packer-plugin-vsphere/post-processor/vsphere-template"
+	vsphereTemplate "github.com/hashicorp/packer-plugin-vsphere/post-processor/vsphere-template"
```

Comment:
For package aliases I don't know that camelCase is preferred tbh, but I don't have a reason to reject this, so that's a LGTM for me!

Reply:
I'm picky. ha!!